tap_sftp/client.py

```diff
@@ -12,10 +13,13 @@
 from datetime import datetime
 from paramiko.ssh_exception import AuthenticationException, SSHException
 
+# set default timeout to 300 seconds
+REQUEST_TIMEOUT = 300
+
 LOGGER = singer.get_logger()
 
 class SFTPConnection():
-    def __init__(self, host, username, password=None, private_key_file=None, port=None):
+    def __init__(self, host, username, password=None, private_key_file=None, port=None, timeout=None):
```

Comment:
@harshpatel4crest Why timeout = None? You can set it to REQUEST_TIMEOUT instead to denote that default value will be taken if not provided

Reply:
Added REQUEST_TIMEOUT instead of None.